src/status_im/contexts/chat/messenger/messages/content/view.cljs

```diff
@@ -126,14 +126,62 @@
       :photo-path          photo-path
       :incoming?           (not= public-key from)}]))
 
+(defn group-member
+  [literal]
+  (let [[primary-name _] (rf/sub [:contacts/contact-two-names-by-identity literal])
+        photo-path       (rf/sub [:chats/photo-path literal])]
+    [rn/view {:style style/group-member-mention}
+     [quo/user-avatar
+      {:full-name         primary-name
+       :profile-picture   photo-path
+       :status-indicator? false
+       :size              :xxxs
+       :ring?             false}]
+     [quo/text
+      {:weight :semi-bold
+       :style  {:margin-left 4}}
+      primary-name]]))
+
+
+(defn resolve-group-system-message
+  [{:keys [^js children]} acc]
```

Comment:
I had the impression `(:parsed-text content)` returned CLJ data, in which case `children` here shouldn't have the `^js` type hint.